src/blueapi/BlueapiComponents.tsx

```diff
@@ -47,6 +48,10 @@ export function RunPlanButton(props: RunPlanButtonProps) {
   const params = props.planParams ? props.planParams : {};
   const variant = props.btnVariant ? props.btnVariant : "outlined";
   const size = props.btnSize ? props.btnSize : "medium";
+  const color = props.btnColor ? props.btnColor : "primary";
+  const disabled = props.disabled ? props.disabled : false;
+  const sx = props.sx ? { ...buttonStyles, ...props.sx } : buttonStyles; // Style for the button component which is the most likely to be customised
```

Comment:
```suggestion
  const sx = props.sx ? { ...buttonStyles, ...props.sx } : {}; // Style for the button component which is the most likely to be customised
```

Leave sx empty if not passed and export the ButtonStyles you set up so they can be called as needed (or overridden as the case may be)